site/content/docs/howto/configure-supervisor-with-workspace_one_access.md

```diff
@@ -117,6 +118,10 @@ stringData:
   clientSecret: "<your-client-secret>"
 ```
 
+Note that the `metadata.name` of the OIDCIdentityProvider resource may be visible to end users at login prompts
+if you choose to enable `allowPasswordGrant`, so choose a name which will be understood by your end users.
+For example, if you work at Acme Corp, choose something like `acme-corporate-workspace-one` over `my-idp`.
```

Comment:
WS1 does not support password grant.

Reply:
Really? That's surprising. Good catch. Removed this paragraph in 39fd9ba2701477db1f8b1032e1fb6fe8c8f87a15.